### ISSUE TITLE
In the latest version, nested folders are not recognized by the game

### ISSUE THREAD

**Issue by beixinti** [2394425147/astrodx#115]
## 最新版本中，嵌套的文件夹无法被游戏识别到

在过去的版本中，以下格式存放的谱面文件可以被游戏识别，它们嵌套了一层分类文件夹：
/storage/emulated/0/Android/data/com.Reflektone.AstroDX/files/levels/class/level/

但是在最新测试版本中，只有以以下方式存放的文件会被识别：
/storage/emulated/0/Android/data/com.Reflektone.AstroDX/files/levels/level/

也就是说，在最新测试版中，无法将谱面文件分类，所有谱面都只能存放在 All 文件夹中。这大大降低了游玩体验🥲

我并不清楚这是 Bug 还是现版本存在的问题，如果此问题是已经存在的问题，并且已经在新版本解决了，请评论版本号并**帮助我关闭此 issue** (因为我并不是经常访问 Github) 🙏

> 游戏版本号：v2.0.0 Beta 1 (Patch 4)
> 设备：联想小新 Pad 2024 (TB331FC)
> 安卓版本：Android 13
> 系统版本：ZUI 15.1.328 (最新稳定版)
---
## In the latest version, nested folders are not recognized by the game
In the old versions, the chart files stored in the following format can be recognized by gmae. They nested a classification folder:
/storage/emulated/0/Android/data/com.Reflektone.AstroDX/files/levels/class/level/

However, in the latest development version, only chart files stored in the following ways will be identified:
/storage/emulated/0/Android/data/com.Reflektone.AstroDX/files/levels/level/

In other words, in the latest development version, the chart file cannot be classified, and all the spectrum can only be stored in the ALL folder. This greatly reduces the play experience🥲

I don't know if this is a problem with the existence of the bug or the current version. If this problem is already a problem existing, and it has been resolved in the new version, please comment on the version number and **help me close this issue** (because I do not visit github frequently) 🙏

> Game Version：v2.0.0 Beta 1 (Patch 4)
> Device：Lenovo Tab M11 (TB331FC)
> Android Version：Android 13
> OS Version：ZUI 15.1.328 (Latest stable version)

**Comment by 2394425147** [comment]
你好！
新版的谱面存储方式确实只能通过单层文件夹读取，收藏夹信息存储在位于collection文件夹下的各个manifest里面，这样做是为了允许多个收藏夹收藏同一个谱面，然后也方便每个收藏夹添加额外信息，以后可以根据更新链接自动下载谱面，等等
目前，beta 1 patch 3及更新版本支持通过.adx自动导入，具体操作方式可以参考https://sht.moe/adx-android 的 Using 2.0 Beta 部分
感谢！

Hi!
This was an intentional design change, and you'll be able to find information on your collections inside manifest.json files located inside the collections folder. This was done to allow multiple collections to reference the same level, and also allows collections to possess more data, which enables us to make auto-updatable collections via urls, and much more.
All versions after (and including) Beta 1 Patch 3 should support importing levels through adx archives. You can learn more about it [here](https://sht.moe/adx-android), in the Using 2.0 Beta section.
Thanks for the issue!

**Comment by 2394425147** [comment]
因为这个正好是新版本在测试的内容，所以为了避免重复issue，现在先置顶了👍 

Since this is an experimental feature being tested in the new versions (and is not properly documented), I'll pin this issue for now to avoid duplicate submissions, thanks 👍

**Comment by Gennadiyev** [comment]
I think this issue should be in closed status, whilst being sticked to the top 😺

**Comment by ts8zs** [comment]
我觉得可以两种收藏方式并存，同时保留
I think both collection methods can coexist while retaining
